CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## future release (TBD)
 Bugfixes:
 * Dont use SocketCAN hardware timestamp as default but software timestamp. Hardware timestamp not being a unix epoch timestamp leads to problems.
+Improvements:
+* Added Mac-user-friendly commands
```

Comment:
This change is only for customer demo, right? Shall we say **Added Mac-user-friendly commands in quick demo**